workflow-testing/src/main/java/com/squareup/workflow/testing/RealRenderTester.kt

```diff
@@ -105,6 +130,12 @@ internal class RealRenderTester<PropsT, StateT, OutputT, RenderingT>(
     return this
   }
 
+  // TODO unit tests
+  override fun disallowUnexpectedChildren(): RenderTester<PropsT, StateT, OutputT, RenderingT> {
```

Comment:
@rjrjr @bencochran @dhavalshreyas  Thoughts about the name for this? Maybe it should be something like `expectNoOtherChildren`?